tests/cc/test_brb.py

```diff
@@ -0,0 +1,232 @@
+        b = BPF(src_file=arg1, debug=0)
+        pem_fn = b.load_func("pem", BPF.SCHED_CLS)
+        br1_fn = b.load_func("br1", BPF.SCHED_CLS)
+        br2_fn = b.load_func("br2", BPF.SCHED_CLS)
+        ip = IPRoute()
+
+        # program id
+        prog_id_pem = 1
+        prog_id_br1 = 2
+        prog_id_br2 = 3
+
+        # initial port id and table pointers
+        curr_pem_pid = 0
+        curr_br1_pid = 0
+        curr_br2_pid = 0
+        self.get_table(b)
+
+        # configure jump table
+        self.jump.update(c_uint(prog_id_pem), c_uint(pem_fn.fd))
+        self.jump.update(c_uint(prog_id_br1), c_uint(br1_fn.fd))
+        self.jump.update(c_uint(prog_id_br2), c_uint(br2_fn.fd))
+
+        # connect pem and br1
+        curr_pem_pid = curr_pem_pid + 1
+        curr_br1_pid = curr_br1_pid + 1
+        self.connect_ports(prog_id_pem, prog_id_br1, curr_pem_pid, curr_br1_pid,
+                      ip, self.br1_dest, self.br1_mac,
+                      self.ns1_eth_out, self.vm1_mac, self.vm1_ip)
+
+        # connect pem and br2
+        curr_pem_pid = curr_pem_pid + 1
+        curr_br2_pid = curr_br2_pid + 1
+        self.connect_ports(prog_id_pem, prog_id_br2, curr_pem_pid, curr_br2_pid,
+                      ip, self.br2_dest, self.br2_mac,
+                      self.ns2_eth_out, self.vm2_mac, self.vm2_ip)
+
+        # connect <br1, rtr> and <br2, rtr>
+        ifindex = ip.link_lookup(ifname=self.nsrtr_eth0_out)[0]
+        self.br1_rtr.update(c_uint(0), c_uint(ifindex))
+        ifindex = ip.link_lookup(ifname=self.nsrtr_eth1_out)[0]
+        self.br2_rtr.update(c_uint(0), c_uint(ifindex))
+
+        # tc filter setup with bpf programs attached
+        self.attach_filter(ip, self.ns1_eth_out, pem_fn.fd, pem_fn.name)
+        self.attach_filter(ip, self.ns2_eth_out, pem_fn.fd, pem_fn.name)
+        self.attach_filter(ip, self.nsrtr_eth0_out, br1_fn.fd, br1_fn.name)
+        self.attach_filter(ip, self.nsrtr_eth1_out, br2_fn.fd, br2_fn.name)
+
+    def setUp(self):
+
+        # set up the environment
+        self.set_default_const()
+        self.setup_vm_ns(self.ns1, self.ns1_eth_in, self.ns1_eth_out)
+        self.setup_vm_ns(self.ns2, self.ns2_eth_in, self.ns2_eth_out)
+        self.config_vm_ns(self.ns1, self.vm1_ip, self.vm2_rtr_mask, self.vm1_rtr_ip)
+        self.config_vm_ns(self.ns2, self.vm2_ip, self.vm1_rtr_mask, self.vm2_rtr_ip)
+        self.setup_router_ns(self.ns_router, self.nsrtr_eth0_in, self.nsrtr_eth0_out,
+                             self.nsrtr_eth1_in, self.nsrtr_eth1_out)
+        self.config_router_ns(self.ns_router, self.vm1_rtr_ip, self.vm2_rtr_ip)
+
+        # get vm mac address
+        self.vm1_mac = commands.getoutput('ip netns exec ' + self.ns1 + ' cat /sys/class/net/eth0/address')
+        self.vm2_mac = commands.getoutput('ip netns exec ' + self.ns2 + ' cat /sys/class/net/eth0/address')
+
+        # load the program and configure maps
+        self.config_maps()
+
+    def test_brb(self):
```

Comment:
may be include example 'make test' output in commit log and/or this .py file, so users know what to expect?